v5_MIGRATION_GUIDE.md

```diff
@@ -86,9 +86,9 @@ class MyActivity : FragmentActivity() {
     
     fun onVenmoButtonClick() {
 -       venmoClient.tokenizeVenmoAccount(activity, request)
-+       venmoClient.requestAuthChallenge(this, venmoRequest) { authChallenge, error ->
++       venmoClient.requestAuthChallenge(this, venmoRequest) { paymentAuthRequest, error ->
 +            error?.let { /* handle error */ }
-+            authChallenge?.let { venmoLauncher.launch(it) }
++            paymentAuthRequest?.let { venmoLauncher.launch(paymentAuthRequest) }
```

Comment:
```suggestion
+            paymentAuthRequest?.let { venmoLauncher.launch(it) }
```
nit: Unless `paymentAuthRequest` gets smart unwrapped?

Reply:
Yeah this is going to change when we implement the single result object but updated for now 👍